### PR TITLE
Update CentosBase to AlmaLinux as Centos is EOL

### PR DESCRIPTION
**Pull request by raunakkumar**
https://www.centos.org/centos-linux-eol/

Signed-off-by: Raunak <raunak.kumar@hpe.com>

### CODE REVIEW

File: cmd/csi-driver/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 RUN microdnf update -y && rm -rf /var/cache/yum
-ADD CentOS-Base.repo /etc/yum.repos.d/
+ADD AlmaLinux-Base.repo /etc/yum.repos.d/
 RUN microdnf install -y systemd && microdnf install -y cryptsetup
 
 LABEL name="HPE CSI Driver for Kubernetes" \
       maintainer="HPE Storage" \
       vendor="HPE" \
-      version="2.0.0" \
+      version="2.1.1.0" \
```
Comment (datamattsson) [inline cmd/csi-driver/Dockerfile:10]: Is this a new convention with four positions?

**Review by datamattsson — APPROVED** [review]
LGTM with questions.

File: cmd/csi-driver/Dockerfile (hunk 1 above)
Comment (raunakkumar) [reply cmd/csi-driver/Dockerfile:10]: Yes, agreed internally with the team unless it brings up any issues. We will know more once we work on the beta helm chart

Comment (datamattsson) [reply cmd/csi-driver/Dockerfile:10]: If the 4th position is going to be package version, is the container considered the package in this instance then? I think in this case the "version" will be the visible "application version" in the partner ecosystems.

Comment (raunakkumar) [reply cmd/csi-driver/Dockerfile:10]: So should we go with application version 2.1.1 and package version 2.1.1.0 ?

**Comment by datamattsson** [comment]
That would make more sense to me.

**Pull request merged**